targets/unity-v2/source/Shared/Internal/OneDsUtility.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Net.Http;
```

Comment:
Please remove this using System.Net.Http.
I know this may work in more recent Unity 2018, but in Unity 2017 (with 4.6 runtime) it doesn't support out-of-the box. Please see another comment below.